api/src/main/java/jakarta/servlet/SessionCookieConfig.java

```diff
@@ -236,4 +238,37 @@
      * @see jakarta.servlet.http.Cookie#getMaxAge
      */
     public int getMaxAge();
+
+    /**
+     * Sets the SameSite mode for the session tracking cookies created on behalf of the application represented
+     * by the <tt>ServletContext</tt> from which this <tt>SessionCookieConfig</tt> was acquired.
+     *
+     * @param sameSiteMode the <i>SameSite</i> mode of the session tracking cookies created on behalf of the application
+     *                     represented by the <tt>ServletContext</tt> from which this <tt>SessionCookieConfig</tt> was
+     *                     acquired. May be null to imply that the SameSite attribute should not be set.
+     *
+     * @throws IllegalStateException if the <tt>ServletContext</tt> from which this <tt>SessionCookieConfig</tt> was
+     *                               acquired has already been initialized
+     *
+     * @see jakarta.servlet.http.Cookie#setSameSiteMode(SameSiteMode)
+     *
+     * @since Servlet 4.x
+     */
+    void setSameSiteMode(SameSiteMode sameSiteMode);
```

Comment:
I prefer `setSameSite(SameSite sameSite);`  ie drop the Mode part of the name.

Reply:
done